README.md

```diff
@@ -1,2 +1,127 @@
-# erik
-Everything you need to start running your client Jasmine tests headlessly with PhantomJS.
+# Erik
+
+Start running your client `Jasmine` tests headlessly with `PhantomJS` and `gulp` today.
+
+### When should I use Erik?
+
+You should use Erik when you have a suite of client Jasmine tests that you currently run by opening a `SpecRunner.html` file but would prefer to run headlessly via a `gulp` task.
+
+Erik utilizes [Karma](https://github.com/karma-runner/karma) to run your Jasmine tests with PhantomJS.
+
+It abstracts away Karma's details and configuration so that you can begin running your tests with PhantomJS quickly and easily.
+
+Simply adapt the below example configuration for your use.
+
+### Installation
+
+`$ yarn add -D erik`
+
+or
+
+`$ npm install --save-dev erik`
+
+### Can I use this with `my CI platform`?
+
+Yes! Erik's exit code is determined by the outcome of your test suite, so integration with your CI platform should be easy.
+
+### Notes on our assumed configuration (or, "How we make it easy for you")
+
+##### Fetching remote dependencies
+
+We use [`gulp-remote-src`](https://github.com/ddliu/gulp-remote-src) to fetch the remote dependencies that currently live in your `SpecRunner.html` `script` tags.
+
+##### No Karma-configured pre-processing
+
+If you're like [us](https://github.com/mixmaxhq), you already have `gulp` tasks for bundling (and perhaps transpiling) your test suite along with its dependencies. It's easier to keep that configuration and simply pass the path of your bundled, transpiled test suite than it is to duplicate that configuration with Karma preprocessing. Additionally, we've disabled Karma's default preprocessing of `coffeescript` files.
```

Comment:
So, I like this philosophy, but why go out of our way to enforce it like disabling the default Coffeescript preprocessing? It complicates our configuration, so unless it slows down Karma I'd leave it enabled.